docs/user/welcome/architecture.rst

```diff
@@ -69,17 +70,13 @@ system authorities and so on.
 +---------------------+------------------------+--------------------------------------+
 |                     | ``gt-jdbc-terasdata``  | Teradata                             |
```

Comment:
```suggestion
|                     | ``gt-jdbc-teradata``   | Teradata                             |
```

Reply:
actually removing as that one is unsupported